dir.yaml

```
@@ -186,7 +186,7 @@
     # - data-integration/data-bridge-cassa
     - data-integration/data-bridge-tdengine
     - data-integration/data-bridge-rocketmq
-    
+    - data-integration/data-bridge-sqlserver
```

We need to comment it util e5.0.3 released
```suggestion
    #- data-integration/data-bridge-sqlserver
```